src/main/java/org/kuali/kra/proposaldevelopment/bo/ProposalAbstract.java

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
This file was included in the commit as a new file but I don't see where the changes are in the code...